core/src/main/java/com/yahoo/oak/EntrySet.java

```diff
@@ -190,78 +190,61 @@ int getLastEntryIndex() {
      * Converts external entry-index to internal array index.
      * <p>
      * We use the following terminology:
-     *  - intIdx for the integer's index inside the entries array (referred as a set of integers)
+     *  - longIdx for the long's index inside the entries array (referred as a set of longs)
      *  - entryIdx for the index of entry array (referred as a set of entries)
      *
      * @param entryIdx external entry-index
      * @return the internal array index
      */
-    private static int entryIdx2intIdx(int entryIdx) {
+    private static int entryIdx2LongIdx(int entryIdx) {
         if (entryIdx == 0) { // assumed it is a head pointer access
             return 0;
         }
         return ((entryIdx - 1) * FIELDS) + HEAD_NEXT_INDEX_SIZE;
     }
 
     /**
+<<<<<<< HEAD
+     * getEntryArrayFieldLong atomically reads long field of the entries array.
+     * Could be used with any OFFSET.value
+=======
      * getEntryArrayFieldInt gets the integer field of entry at specified offset for given
      * start of the entry index in the entry array.
      * The field is read atomically as it is a machine word, however the concurrency of the
      * mostly updated value is not ensured as no memory fence is issued.
      */
-    private int getEntryArrayFieldInt(int intFieldIdx, OFFSET offset) {
+    private long getEntryArrayFieldInt(int intFieldIdx, OFFSET offset) {
         return entries[intFieldIdx + offset.value]; // used for NEXT
     }
```

Comment:
It is currently not in use. I suggest using it when an int field is required.
```suggestion
    private long getEntryArrayFieldInt(int longStartFieldIdx, OFFSET offset) {
        return entries[intFieldIdx + offset.value] & ((1L << 32) - 1); // used for NEXT
    }
```

Reply:
I do not think it worth to keep a function that is not in use. getEntryArrayFieldInt removed for now